.github/labeler.yml

```diff
@@ -0,0 +1,38 @@
+# Configuration for GitHub Action `.github/workflows/pull-request-labeler.yml`.
+# See https://github.com/marketplace/actions/labeler for more information.
+
+# Only documentation changes
+docs:
+  - all: [ '**/*.md' ]
+
+# Only test changes
+dev/testing:
+  - all: [ '{spec/**,integration/**}' ]
+
+# Changes to Profiling
+profiling:
+  - any: [ 'lib/datadog/profiling/**' ]
+
+# Changes to CI-App
+ci-app:
+  - any: [ 'lib/datadog/ci/**' ]
+
+# Changes to ASM
+appsec:
+  - any: [ 'lib/datadog/appsec/**' ]
+
+# Changes to Tracing integrations
+integrations:
+  - any: [ 'lib/datadog/tracing/contrib/**' ]
```

Comment:
Should there be `'lib/datadog/appsec/contrib/**'` here too? Maybe `'lib/datadog/ci/contrib/**' as well?

Reply:
Also should there be a general tracing one as well?

```
tracing:
  - any: [ 'lib/datadog/tracing/**' ]
```

Reply:
Not sure what the behaviour is when multiple criteria apply (e.g resulting in `AppSec`+`Integrations` tags, which sounds good to me if someone changes AppSec integration code).

Reply:
Also this non-product one sounds OK to me as well:

```
core:
  - any: [ 'lib/datadog/core/**' ]
```

Reply:
Applied all 3 suggested changes.